### PR TITLE
add notification settings

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

File: src/page/settings/view.js [hunk 1]
```diff
@@ -88,6 +105,67 @@ class SettingsPage extends React.PureComponent {
             </View>
           </View>
 
+          <View style={settingsStyle.sectionDivider} />
+          <Text style={settingsStyle.sectionTitle}>{__('Notifications')}</Text>
+          <Text style={settingsStyle.sectionDescription}>
+            {__('Choose the notifications you would like to receive.')}
+          </Text>
+          <View style={settingsStyle.row}>
```
Comment (kauffj) [inline src/page/settings/view.js]: there's a lot of copy/paste here

File: src/page/settings/view.js [hunk 2]
```diff
@@ -88,6 +105,67 @@ class SettingsPage extends React.PureComponent {
...
+            <View style={settingsStyle.switchText}>
+              <Text style={settingsStyle.label}>{__('Subscribed content')}</Text>
```
Comment (kauffj) [inline src/page/settings/view.js]: Subscriptions

File: src/page/settings/view.js [hunk 3]
```diff
@@ -88,6 +105,67 @@ class SettingsPage extends React.PureComponent {
...
+                onValueChange={value => {
+                  this.setNativeBooleanSetting(SETTINGS.RECEIVE_REWARD_NOTIFICATIONS, value);
+                }}
+              />
+            </View>
+          </View>
+
+          <View style={settingsStyle.row}>
+            <View style={settingsStyle.switchText}>
+              <Text style={settingsStyle.label}>{__('Your interests')}</Text>
```
Comment (kauffj) [inline src/page/settings/view.js]: Tags you follow

File: src/page/settings/view.js [hunk 4]
```diff
@@ -88,6 +105,67 @@ class SettingsPage extends React.PureComponent {
...
+                onValueChange={value => {
+                  this.setNativeBooleanSetting(SETTINGS.RECEIVE_INTERESTS_NOTIFICATIONS, value);
+                }}
+              />
+            </View>
+          </View>
+
+          <View style={settingsStyle.row}>
+            <View style={settingsStyle.switchText}>
+              <Text style={settingsStyle.label}>{__('Content creator tips')}</Text>
```
Comment (kauffj) [inline src/page/settings/view.js]: Don't add this one yet

File: src/page/settings/view.js (hunk 1 above)
Comment (akinwale) [reply src/page/settings/view.js]: I can create a component for the settings rows.

**Pull request merged**